src/unify_table/unify_table.mbt

```diff
@@ -253,20 +227,11 @@ pub fn[T : Eq] get(self : UnificationTable[T], index : Int) -> VarValue[T]? {
 /// Example:
 ///
 /// ```moonbit
-/// ///|
-/// test "UnificationTable::index" {
-///   let table : UnificationTable[Int] = new(capacity=5)
-///   table.push(42)
-///   inspect!(table.index(0).index, content="0")
-/// }
-/// 
-/// ///|
```

Comment:
This deletion is unfortunate, but according to @bzy-debug there isn't a way to express panicking tests using the new syntax.